Automattic/jetpack
# Stats: Update dashboard widget aesthetics

## Proposed changes:


Before|After
-|-
<img width="544" alt="image" src="https://user-images.githubusercontent.com/4044428/226764725-c34093c6-d929-40e0-a1c1-fdb5c4547d5e.png">|<img width="540" alt="image" src="https://user-images.githubusercontent.com/4044428/227024882-d0e07f82-30bc-4045-b13a-79304a4a374a.png">


- Updates the look and feel of the Jetpack Stats widget on the wp-admin dashboard.
- Removes the link for configuring Jetpack Stats.
- This design is not final, but I recommend shipping this as it's an improvement over the existing widget.

### Other information:


## Jetpack product discussion


Based heavily off of p1HpG7-l7H-p2.

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Apply these changes to your site with the Jetpack plugin.
* Navigate to `/wp-admin/` and ensure that the Jetpack Stats widget renders with the new aesthetics.
